package.json

```diff
@@ -58,6 +58,7 @@
     "onCommand:r.runCommandWithEditorPath",
     "onCommand:r.runCommand",
     "onCommand:r.showHelp",
+    "onCommand:editor.action.webvieweditor.showFind",
```

Comment:
I think this should be removed, since we don't want the extension to activate on any `showFind` command and if the find command is meant for a help-panel-webview, the extension will already be active anyways

Reply:
oops, fixed (9a5705d85a533df4bb872433de8a2253a5bca70d)